packages/cli/generation/ir-migrations/src/migrations/v38-to-v37/__test__/__snapshots__/migrateFromV38ToV37.test.ts.snap

```diff
@@ -175,293 +175,7 @@ exports[`migrateFromV38ToV37 snapshot 1`] = `
               },
               "response": {
                 "_visit": [Function],
-                "body": {
```

Comment:
I'm accepting this snapshot update because this is how the V44->V43 migrator is meant to behave:
- keep only response body examples
- wipe away stream and sse response examples

The v38toV37 migrator previously generated a dummy example from the streaming endpoint, which is being treated as a body JSON example, which is incorrect.